scripts/un/sdg/README.md

```diff
@@ -23,7 +29,9 @@ Produces:
   * unit.mcf
 * csv/ folder: 
   * [CODE].csv
-(Note that the `schema/` folder is not included in the repository but can be regenerated by running the script.)
+(Note that these folders are not included in the repository but can be regenerated by running the script.)
+
+When refreshing the data, the `geography`, `schema`, and `csv` folders might all get updated and will need to be resubmitted to g3. The corresponding TMCF file is `sdg.tmcf`.
 
 To run unit tests: 
```

Comment:
Please consider adding caveats / gotchas to the README for context / follow-ups (like the fact that we drop some series)

Reply:
done